pos/app/routes/_header.master.tsx

```diff
@@ -14,6 +25,19 @@ export const meta: MetaFunction = () => {
 };
 
 export default function FielsOfMaster() {
+  const submit = useSubmit();
+  const mutateOrder = useCallback(
+    (servedOrder: OrderEntity, descComment: string) => {
+      const order = servedOrder.clone();
+      order.addComment("master", descComment);
+      submit(
+        { servedOrder: JSON.stringify(order.toOrder()) },
+        { method: "PUT" },
+      );
+    },
+    [submit],
+  );
```

Comment:
ここはシンプルにしたいですね。
あと提供ではないので、とりあえずそのまま order にしますね
```suggestion
  const mutateOrder = useCallback(
    (order: OrderEntity) => {
      submit(
        { order: JSON.stringify(order.toOrder()) },
        { method: "PUT" },
      );
    },
    [submit],
  );
```